drivers/all_neighbors_system_dashboard/app/models/all_neighbors_system_dashboard/event.rb

```diff
@@ -10,10 +10,6 @@ class Event < GrdaWarehouseBase
 
     belongs_to :enrollment
 
-    def client_id
-      enrollment.source_enrollment.client.id
-    end
```

Comment:
So this was an `integer` but below has been replaced with `personal_id`

Reply:
Yeah -- there is a bit of history here -- they all began life as warehouse IDs, and then it turned out they needed to match the sending system, and they wanted all of the events, not just the ones attached to the included enrollments, so the original decision to associate the events w/ their enrollments got removed, but this code didn't.